apps/site/assets/package.json

```diff
@@ -94,7 +94,7 @@
     "stylelint-order": "^4.0.0",
     "stylelint-scss": "^3.14.2",
     "stylelint-selector-bem-pattern": "^2.1.0",
-    "stylelint-selector-no-mergeable": "git+ssh://git@github.com/nkingsy/stylelint-selector-no-mergeable.git",
+    "stylelint-selector-no-mergeable": "git+https://git@github.com/nkingsy/stylelint-selector-no-mergeable.git",
```

Comment:
You can configure this on your local machine, so that you don't need to change the configuration: `git config --global url."https://github.com/".insteadOf git@github.com:`